packages/flutter/lib/src/cupertino/button.dart

```diff
@@ -224,6 +244,21 @@ class _CupertinoButtonState extends State<CupertinoButton> with SingleTickerProv
 
     final TextStyle textStyle = themeData.textTheme.textStyle.copyWith(color: foregroundColor);
 
+    final BoxConstraints boxConstraints;
+    
+    if (widget.minSize == null && widget.minHeight == null && widget.minWidth == null) {
+      boxConstraints = const BoxConstraints();
```

Comment:
We're changing the default value here right? From kMinInteractiveDimensionCupertino to 0 now. See https://flutter-gold.skia.org/search?issue=69285&crs=github&query=source_type%3Dflutter for the golden diff.